src/Seq.App.Slack/SlackApi.cs

```diff
@@ -1,24 +1,48 @@
-﻿using System.Net.Http;
+﻿using System.Net;
+using System.Net.Http;
 using System.Text;
 using Newtonsoft.Json;
 
 namespace Seq.App.Slack
 {
-    static class SlackApi
+    class SlackApi
     {
-        private static readonly HttpClient HttpClient = new HttpClient();
+        private HttpClient httpClient;
 
-        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings
```

Comment:
Any reason for the change in serializer settings scope here?

Reply:
no good reason, I've reverted it